### PR TITLE
feat: provide output config options [#5]

### PR DESCRIPTION
**Pull request by boostvolt**
### Tasks

- [x] I've read the [CONTRIBUTING](./CONTRIBUTING) guide
- [x] Necessary tests have been added

- Add new config options:
  - `outFilename`
  - `outFormats`
  - `includeWellKnown`
- `.well-known` is now created in the `dist` root and no longer in the `outDir`

### CODE REVIEW

File: src/options.ts [hunk 1]
```diff
@@ -15,6 +15,14 @@ export interface RollupPluginSbomOptions {
...
      */
     outDir?: string;
+    /**
+     * The base filename for the SBOM files, defaults to 'bom'
+     */
+    outFilename?: string;
+    /**
+     * The formats to output, defaults to ['json', 'xml']
+     */
+    outFormats?: string[];
```
Comment (janbiasi) [inline src/options.ts:25]: I'd add a concrete union type for the valid values instead of a string array to prevent any errors within TS itself.

**Review by janbiasi — COMMENTED** [review]
Thanks a lot for your effort! Only a small thing, please take a look below.

File: src/options.ts (hunk 1 above)
Comment (boostvolt) [reply src/options.ts]: sure.

**Review by janbiasi — APPROVED** [review]
Thanks a lot! 🙏🏽

**Pull request merged**